<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PGK - Zadanie #12: Morphing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frameRight">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>520</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj wszystko</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>550</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zako≈Ñcz</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameResult">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="imgLeftButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>520</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj obraz 1...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imgRightButton">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>520</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj obraz 2...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>550</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="framesSpinBox">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>520</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1155</x>
      <y>525</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba klatek</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetLButton">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>520</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetRButton">
    <property name="geometry">
     <rect>
      <x>775</x>
      <y>520</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
